src/plugins/in-app-menu/renderer/TitleBar.tsx

```
@@ -19,6 +19,55 @@ import { PanelItem } from './PanelItem';
+  });
+
+  const handleClick = () => {
+    window.dispatchEvent(new CustomEvent('ytmd-download-manager-toggle'));
+  };
+
+  return (
+    <button
+      class="ytmd-dm-title-btn"
+      onClick={handleClick}
+      title="Gestor de Descargas"
+      data-active={isOpen()}
```

ESLint

[error] Props should be sorted alphabetically

(stylistic/jsx-sort-props)